app/src/main/java/com/google/android/samples/socialite/ui/navigation/SocialiteNavigation.kt

```diff
@@ -28,98 +29,99 @@ import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteScaffo
 import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteScaffoldDefaults
 import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteType
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.res.stringResource
-import androidx.navigation.NavController
-import androidx.navigation.NavDestination
-import androidx.navigation.NavDestination.Companion.hasRoute
-import androidx.navigation.NavDestination.Companion.hierarchy
-import androidx.navigation.NavGraph.Companion.findStartDestination
-import androidx.navigation.compose.currentBackStackEntryAsState
 import com.google.android.samples.socialite.R
 import com.google.android.samples.socialite.ui.navigation.TopLevelDestination.Companion.isTopLevel
+import kotlinx.parcelize.Parcelize
 import kotlinx.serialization.Serializable
 
-sealed interface Route {
+sealed interface Screen : Parcelable {
```

Comment:
One naming thing - if we're renaming away from `Route`, I'd prefer to rename to `Pane` instead of `Screen`.

I think `Screen` is a bit overloaded with the physical screen term that implies a "filling up the entire screen" which isn't the case with `ChatsList` and `ChatThread`